### ISSUE TITLE
Issue: Admin LAN's

### ISSUE THREAD

**Issue by LaocheXe** [septor/anteup#55]
![capture](https://user-images.githubusercontent.com/5818022/35291955-9edc0b42-0034-11e8-9623-456b78934c90.JPG)

I don't know what, but something broke

All I did was update via github - and notice all the lan's showing in AnteUP plugin - No other plguins have this issue

**Comment by Moc** [comment]
In `admin_config.php` on line 16 try replacing this: 
```php
e107::lan('anteup', 'admin', true);
```
with
```php
e107::lan('anteup', true);
```

Does this fix the issue?

**Comment by LaocheXe** [comment]
Yes

**Comment by Moc** [comment]
Fixed in d2457035025928ac25f4f350c9fd65ed21e8b9ce
